File: PHP-API-Form/Predict.py
# -*- coding: utf-8 -*-
"""
Created on Tue Oct 12 11:53:12 2021
@author: ialzu
"""


import sys 
import pandas as pd
from sklearn import model_selection
from sklearn.linear_model import LogisticRegression
import pickle
import argparse


import warnings
warnings.filterwarnings("ignore")


parser = argparse.ArgumentParser()
parser.add_argument("AD_Month", type=int, help="Melady ")
parser.add_argument("Lag1", type=int, help="Lag 1")
parser.add_argument("Lag2",  type=int, help="Lag 2")
parser.add_argument("Lag3", type=int, help="Lag 3")
parser.add_argument("Lag4", type=int, help="Lag 4")
parser.add_argument("Lag5", type=int, help="Lag 5")
parser.add_argument("Lag6",  type=int, help="Lag 6")
parser.add_argument("Lag7",  type=int, help="Lag 7")
parser.add_argument("Long_Vacation", type=int, help="Long Vacation")
parser.add_argument("temp_min", type=int, help="Minimum Temperature")




args = sys.argv

filename = 'finalized_model2.sav'
#pickle.dump(model, open(filename, 'wb'))
 
# some time later...
 
# load the model from disk
loaded_model = pickle.load(open(filename, 'rb'))
result = loaded_model.predict(pd.DataFrame(data=[[args[1],args[2],args[3],args[4],args[5],args[6],args[7],args[8],args[9],args[10],args[11],args[12],args[13],args[14],args[15]]]))
print(result[0])

File: PHP-API-Form/Predict.php
<?php


// Use ls command to shell_exec
// function
//$output = shell_exec('python Predict.py 9 1200 1200 1200 1200 1200 1200 1200 1 68');
      
// Display the list of all file
// and directory
//echo "<pre>$output</pre>";
if($_SERVER["REQUEST_METHOD"]=="POST")
{
    $AD_Month = $_POST["AD_Month"];
    $AH_Month = $_POST["AH_Month"];
    $Long_Vacation = $_POST["Long_Vacation"];
    $AD_WeekdayNum = $_POST["AD_WeekdayNum"];
    $Wake_Up = $_POST["Woke_Hour"];
    $temp_min = $_POST["temp_min"];
    $temp_max = $_POST["temp_max"];
    $Lag1 = $_POST["Lag1"];
     $Lag2 = $_POST["Lag2"];
      $Lag3 = $_POST["Lag3"];
       $Lag4 = $_POST["Lag4"];
        $Lag5 = $_POST["Lag5"];
         $Lag6 = $_POST["Lag6"];
          $Lag7 = $_POST["Lag7"];
           $Lag8 = $_POST["Lag8"];
       
           
           
           // build sys.arg string 
           
           $str =  $AD_Month." ".$AH_Month." ".$Long_Vacation." ".$AD_WeekdayNum." ".$Wake_Up." ".$temp_min." ".$temp_max." ".$Lag1." ".$Lag2." ".$Lag3." ".$Lag4." ".$Lag5." ".$Lag6." ".$Lag7." ".$Lag8;
           $str = '/usr/bin/python3 Predict.py '.$str.' 2>&1';
          // echo $str;
            $output = exec($str);
// $output = exec('pip3 install pandas --no-cache-dir 2>&1');
//$output = exec('python3 Predict2.py ');
  
// Display the list of all file
// and directory
echo $output;
           
}


?>